src/main/scala/interplay/PlaySonatypeBase.scala

```diff
@@ -13,6 +14,7 @@ object PlaySonatypeBase extends AutoPlugin {
   override def requires = Sonatype
 
   override def projectSettings = Seq(
+    playBuildPromoteSonatype in ThisBuild := true,
```

Comment:
Adding this here impacts `PlayRootProjectBase` which, for some reason, `requires` `PlaySonatypeBase` even though it's a [`configuration for a Play Root Project that doesn't get published`](https://github.com/playframework/interplay/blob/d058d2c3671dc6ce6d352b6ef50b9b739c9101d2/src/main/scala/interplay/PlayBuildBase.scala#L80-L83). 

As a consequence, the `release` step on the `scripted` tests fails.

I think the correct solution is to apply the change:

```diff
-  override def requires = PlayBuildBase && PlayBintrayBase && PlaySonatypeBase && PlayReleaseBase
+  override def requires = PlayBuildBase && PlayReleaseBase
```

YOLO